qiling/core.py

```diff
@@ -670,7 +688,46 @@ def enable_lib_patch(self):
                 self.mem.write(self.mem.get_lib_base(filename) + addr, code)
             except:
                 raise RuntimeError("Fail to patch %s at address 0x%x" % (filename, addr))
-    
+
+    def _init_stop_guard(self):
```

Comment:
Why not put this function to utils.py?

Reply:
no reason, do you want me to move it there?

Reply:
Since it's also one of the core functions (start/stop etc..), it's okay to leave it here.